README.md

```diff
@@ -41,6 +41,9 @@ The website has the following **mandatory** environment variables
 | :----------------- | :--------------------------------------------- |
 | GHOST_URL          | Your Ghost CMS URL                             |
 | GHOST_API_KEY      | Your Ghost CMS API key                         |
+| GHOST_ALLPOSTS_URL | Your URL for all ghost posts, see below        |
```

Comment:
If we're going to use `GHOST_ALLPOSTS_URL` why not  just remove `GHOST_API_KEY` and `GHOST_URL` since it's required in the "URL" now ? (like postgressql url requiring to fill it up fully instead of manually providing the user and password and port and location and ...)

Another solution would be to not ask for the URL itself and build it internally instead.

TLDR: This feels like a hack, I mean it works but I would appreciate some more effort by bundling the url since you effectively only got 2 variables which are already provided by `GHOST_URL` and `GHOST_API_KEY`.